.travis.yml

```diff
@@ -36,8 +36,6 @@ install:
 before_script:
   - echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
-script:
-  - vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+script: ./vendor/bin/phpunit tests/YahooWeatherClientTests.php --coverage-html --coverage-clover=coverage.xml
```

Comment:
Ok, the PHPUnit does not need ```-c phpunit.xml``` and it can automatically detect the phpunit.xml file.
I just use the command: ```vendor/bin/phpunit```